beacon-chain/sync-gossip.md

```diff
@@ -2,9 +2,9 @@
 >  This specification is a work-in-progress and should be considered preliminary.
 
 ## Overview
-A beacon chain client could sync committee to perform [state updates](https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/sync-protocol.md). The data object [LightClientSnapshot](https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/sync-protocol.md#lightclientsnapshot) allows a client to quickly sync to a particular header. Once the client establishes a valid header, it could sync to other headers by processing [LightClientUpdates](https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/sync-protocol.md#lightclientupdate). These two data types allow a client to stay up-to-date with the beacon chain.
+A beacon chain client could sync committee to perform [state updates](https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/sync-protocol.md). The data object [LightClientSkipSyncUpdate](skip-sync-network) allows a client to quickly sync to a reasonably recent header. Once the client establishes a recent header, it could sync to other headers by processing [LightClientUpdates](https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/sync-protocol.md#lightclientupdate). These two data types allow a client to stay up-to-date with the beacon chain.
```

Comment:
The correct phrasing would be to get the current sync period's sync committee pubkeys. It doesn't matter if you have a header 1min or 23h old as long as you have the right pubkeys to validate incoming headers.

Reply:
Updated the wording.